Dockerfile

```diff
@@ -3,6 +3,7 @@ RUN apk update && apk upgrade && apk add sqlite-static
 WORKDIR /build/
 ARG version
 RUN git clone --branch ${version:-dist} --depth 1 https://github.com/toddsundsted/ktistec .
+RUN shards update
```

Comment:
Without this, I got the error 
```
#13 6.141 Outdated shard.lock (backtracer requirements changed). Please run shards update instead.
```

Reply:
Same error here as well without running `shards update` first.